Dockerfile

```diff
@@ -1,18 +1,17 @@
 FROM registry.access.redhat.com/ubi8/ruby-27
 
 USER root
-RUN dnf --setopt=skip_missing_names_on_install=False,tsflags=nodocs --save \
-  && rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
+RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
   && dnf update -y \
-  && dnf remove -y postgresql \
-  && dnf install -y shared-mime-info postgresql12 postgresql12-devel postgresql12-libs \
+  && dnf remove -y postgresql* \
+  && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs -y shared-mime-info postgresql12-12.13 postgresql12-devel-12.13 postgresql12-libs-12.13 \
   && dnf clean all \
   && rm -rf /var/cache/yum
 
 USER default
 WORKDIR ${APP_ROOT}
 
-RUN gem install bundler --version=2.2.19 --no-document
+RUN gem install bundler --version=2.3.25 --no-document
```

Comment:
That's the version specified in Gemfile.lock

Reply:
```suggestion
RUN BUNDLER_VERSION=$(awk '/BUNDLED WITH/ { getline; print $1 }' Gemfile.lock) \
    gem install bundler --version=$BUNDLER_VERSION --no-document
```

Reply:
Which Gemfile.lock?
https://github.com/3scale/porta/blob/30f3232eee6da0d76a18076261be6bf4e4bb8622/Gemfile.lock#L1068-L1069